menas/src/main/scala/za/co/absa/enceladus/menas/controllers/VersionedModelController.scala

```diff
@@ -98,6 +98,29 @@ abstract class VersionedModelController[C <: VersionedModel with Product with Au
     versionedModelService.getAllVersions(name)
   }
 
+  @GetMapping(Array("/exportItem/{name}/{version}"))
+  @ResponseStatus(HttpStatus.OK)
+  def exportSingleEntity(@PathVariable name: String, @PathVariable version: Int): CompletableFuture[String] = {
+    versionedModelService.exportSingleItem(name, version)
+  }
+
+  @GetMapping(Array("/exportItem/{name}"))
+  @ResponseStatus(HttpStatus.OK)
+  def exportLatestEntity(@PathVariable name: String): CompletableFuture[String] = {
+    versionedModelService.exportLatestItem(name)
+  }
+
+  @PostMapping(Array("/importItem"))
+  @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Is the L[ocation header sent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201) when the item is imported? Is it done somehow behind the scenes or is it missing?

Reply:
It should be behind the scenes